modules/extensions-spring-support/src/test/resources/models/typed-value.json

```diff
@@ -2759,7 +2759,7 @@
           }
         }
       },
-      "minMuleVersion": "4.1.1",
+      "minMuleVersion": "4.5.0",
```

Comment:
Method getAquarium has min mule version 4.5.0 because of its parameter aquarium. Parameter aquarium has min mule version 4.5.0 because it is of type Aquarium. Type Aquarium has min mule version 4.5.0 because of its field pond. Field pond has min mule version 4.5.0 because it is annotated with ParameterGroup. ParameterGroup was introduced in Mule 4.5.0.